docs/screenshot-and-gif-software.md

```diff
@@ -0,0 +1,36 @@
+# Screenshot and GIF software
+
+The following list documents different screenshoting & GIF-making techniques and
+free software. We encourage you to make use of these when making front-end pull
+requests, as other contributors can see the changes you have made without having
+to checkout your branch.
+
+## Screenshot tools by platform
+### Browser 
+- [LightShot Screenshot](https://app.prntscr.com/en/index.html) (Chrome, Firefox, IE & Opera)
+
+### MacOS
+- Command-Shift-3 to capture all of the screen
+- Command-Shift-4 and drag cursor to select a specific area
+- [LightShot Screenshot](https://app.prntscr.com/en/index.html)
+
+### Windows
+- Snipping Tool (inbuilt)
+- [LightShot Screenshot](https://app.prntscr.com/en/index.html)
+
+### Linux
+- Ctrl-Shift-PrtScn for “select an area to grab” feature of gnome-screenshot.
+
+## GIF tools by platform
+### Browser
+- [GIPHY](https://giphy.com)
+
+### MacOS
+- [GIF Brewery](http://gifbrewery.com/)
```

Comment:
Giphy has a native app for Mac with a nice UI. 
I think you can mention that as well.
[Giphy Capture](https://giphy.com/apps/giphycapture)